bindings/python/test/estimator/test_tle_solver.py

```
@@ -90,106 +93,123 @@ def test_access_methods(self, tle_solver: TLESolver):
+        assert (
+            "Initial guess must be a TLE or (State, B*) when also estimating B*."
+            in str(e.value)
         )
 
-    def test_estimate_tle_no_observations(
+    def test_estimate_no_observations(
         self,
         tle_solver: TLESolver,
         initial_state: State,
     ):
         with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)